mlos_storage_demo.ipynb

```diff
@@ -0,0 +1,976 @@
+       " 'MysqlBench2VMs-testifworking': Experiment :: MysqlBench2VMs-testifworking,\n",
+       " 'MysqlBench2VMs.categorical.1': Experiment :: MysqlBench2VMs.categorical.1,\n",
+       " 'MysqlBench2VMsKernel': Experiment :: MysqlBench2VMsKernel,\n",
+       " 'MysqlBench2VMsKernel1': Experiment :: MysqlBench2VMsKernel1,\n",
+       " 'MysqlBench2VMscategorical-1': Experiment :: MysqlBench2VMscategorical-1,\n",
+       " 'MysqlBench2VMscategorical1': Experiment :: MysqlBench2VMscategorical1,\n",
+       " 'RedisBench': Experiment :: RedisBench,\n",
+       " 'RedisBenchBoot': Experiment :: RedisBenchBoot,\n",
+       " 'mysql-flex': Experiment :: mysql-flex,\n",
+       " 'mysql-innodb-Standard_A2_v2-1ac516aa-vm1': Experiment :: mysql-innodb-Standard_A2_v2-1ac516aa-vm1,\n",
+       " 'mysql-innodb-Standard_B2s-0fc5b1fd-vm1': Experiment :: mysql-innodb-Standard_B2s-0fc5b1fd-vm1,\n",
+       " 'mysql-innodb-Standard_B2s-b7573c39-vm1': Experiment :: mysql-innodb-Standard_B2s-b7573c39-vm1,\n",
+       " 'mysql-innodb-Standard_D2s_v3-0533c94f-vm1': Experiment :: mysql-innodb-Standard_D2s_v3-0533c94f-vm1,\n",
+       " 'mysql-innodb-Standard_F2s-4faab354-vm1': Experiment :: mysql-innodb-Standard_F2s-4faab354-vm1,\n",
+       " 'mysql-innodb-bseries-v346-vm1': Experiment :: mysql-innodb-bseries-v346-vm1,\n",
+       " 'mysql-innodb-bseries-v346-vm2': Experiment :: mysql-innodb-bseries-v346-vm2,\n",
+       " 'mysql-innodb-bseries-v346-vmcontrol': Experiment :: mysql-innodb-bseries-v346-vmcontrol,\n",
+       " 'mysql-innodb-bseries2ms-346-bm': Experiment :: mysql-innodb-bseries2ms-346-bm,\n",
+       " 'mysql-innodb-bseries2ms-346-east': Experiment :: mysql-innodb-bseries2ms-346-east,\n",
+       " 'mysql-innodb-bseries2ms-346-east-control': Experiment :: mysql-innodb-bseries2ms-346-east-control,\n",
+       " 'mysql-innodb-bseries2ms-346-vm': Experiment :: mysql-innodb-bseries2ms-346-vm,\n",
+       " 'mysql-innodb-bseries2ms-346-vm-control': Experiment :: mysql-innodb-bseries2ms-346-vm-control,\n",
+       " 'mysql-innodb-buf': Experiment :: mysql-innodb-buf,\n",
+       " 'mysql-innodb-buf-param346test': Experiment :: mysql-innodb-buf-param346test,\n",
+       " 'mysql-innodb-buff-param346test': Experiment :: mysql-innodb-buff-param346test,\n",
+       " 'mysql-innodb-buffedup-346testD2S': Experiment :: mysql-innodb-buffedup-346testD2S,\n",
+       " 'mysql-innodb-control-346testD2S': Experiment :: mysql-innodb-control-346testD2S,\n",
+       " 'mysql-innodb-explore-D2S': Experiment :: mysql-innodb-explore-D2S,\n",
+       " 'mysql-innodb-explore-D2S-1': Experiment :: mysql-innodb-explore-D2S-1,\n",
+       " 'mysql-kernel-hp': Experiment :: mysql-kernel-hp,\n",
+       " 'mysqlflex': Experiment :: mysqlflex,\n",
+       " 'postgres-kernel-hp': Experiment :: postgres-kernel-hp}"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "storage.experiments"
```

Comment:
Suggest we limit the number of experiments listed here?

Reply:
sure - or I can just delete some experiments from sqlite DB